4985_CommAudio/audio.cpp

```diff
@@ -121,9 +120,13 @@ void initAudioOutput()
 void play(QString filePath)
 {
     currTrack = filePath; // set current track to what user selected
-    loadAudioData(currTrack); // load raw data from wav file to a buffer
-    //loadAudioStream(); ** implement later **
-    playStream();
+
+    if (prevTrack != currTrack || output->state() == QAudio::IdleState && prevTrack == filePath)
```

Comment:
Not something you necessarily have to change, I just wanna confirm logic.  Should it be `(A || B && C)` or something like `((A || B) && C)` or `(A || (B && C))`?